### PR TITLE
Jarv/nginx redirect

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: playbooks/roles/aws/tasks/main.yml [hunk 1]
```diff
@@ -49,10 +49,14 @@
 # Remove them if they are regular files
 # This can be removed once we don't need to worry
 # about backwards compatibility.
-- stat: path={{ COMMON_BIN_DIR }}/{{ aws_s3_sync_script|basename }}
+- local_action:
```
Comment (feanil) [inline playbooks/roles/aws/tasks/main.yml]: Why are these local actions, shouldn't they happen on the server doing the syncing?

Comment (jarv) [inline playbooks/roles/aws/tasks/main.yml]: they come right before the copy, which is copying a file from the local system to the remote.

Comment (feanil) [inline playbooks/roles/aws/tasks/main.yml]: That seems true for the SSL cert related ones, but doesn't make sense here for the s3_sync_script.  The {{COMMON_BIN_DIR}} path isn't going to exist on the local system.

Comment (jarv) [inline playbooks/roles/aws/tasks/main.yml]: you are right, updated.

**Comment by feanil** [comment]
:+1:

**Comment by hkim823** [comment]
@jarv still want this to be merged?

**Comment by jarv** [comment]
These changes were made in another branch, closing

**Pull request closed without merging**